tests/core/search_results_viewed/user_viewed_search_results/statements.json

```diff
@@ -0,0 +1,53 @@
+[
+  {
+    "actor": {
+      "account": {
+        "homePage": "http://www.example.org",
+        "name": "1"
+      },
+      "name": "test_fullname"
+    },
+    "verb": {
+      "id": "https://w3id.org/xapi/acrossx/verbs/searched",
+      "display": {
+        "en": "Searched"
+      }
+    },
+    "object": {
+      "id": "http://www.example.org/search/index.php",
+      "objectType": "Activity",
+      "definition": {
+        "type": "https://w3id.org/xapi/acrossx/activities/webpage",
+        "name": {
+          "en": "Global Search"
+        }
+      }
+    },
+    "result": {
+      "response": "query"
```

Comment:
i presume this is because you hardcoded the query but it would normally contain the actual query?